tools/gen_openvino_dockerfile.py

```diff
@@ -108,12 +108,23 @@ def dockerfile_for_linux(output_file):
 WORKDIR /opt/openvino
 RUN cp -r /workspace/openvino/licensing LICENSE.openvino
 RUN mkdir -p include && \
-    cp -r /workspace/install/runtime/include/ie/* include/. && \
     cp -r /workspace/install/runtime/include/ngraph include/. && \
     cp -r /workspace/install/runtime/include/openvino include/.
 RUN mkdir -p lib && \
-    cp /workspace/install/runtime/lib/intel64/*.so lib/. && \
-    cp /workspace/install/runtime/3rdparty/omp/lib/libiomp5.so lib/.
+    cp /workspace/install/runtime/lib/intel64/libopenvino.so.${OPENVINO_VERSION} lib/. && \
+    cp /workspace/install/runtime/lib/intel64/libopenvino_c.so.${OPENVINO_VERSION} lib/. && \
+    cp /workspace/install/runtime/lib/intel64/libopenvino_intel_cpu_plugin.so lib/. && \
+    cp /workspace/install/runtime/lib/intel64/libopenvino_ir_frontend.so.${OPENVINO_VERSION} lib/.
+RUN OV_SHORT_VERSION=`echo ${OPENVINO_VERSION} | awk '{print substr($0,3,2)}'` && \
+    OV_SHORT_VERSION=${OV_SHORT_VERSION}`echo ${OPENVINO_VERSION} | awk '{print substr($0,6,1)}'` && \
+    OV_SHORT_VERSION=${OV_SHORT_VERSION}`echo ${OPENVINO_VERSION} | awk '{print substr($0,8,1)}'` && \
```

Comment:
See [this comment](https://github.com/triton-inference-server/onnxruntime_backend/pull/200/files#r1254952910)

Reply:
Updated: [Improve OV short version generation and link so files directly](https://github.com/triton-inference-server/openvino_backend/pull/57/commits/d4de999ace41ad368f66084ad25373d7cf3dee30)